department.rb

```diff
@@ -0,0 +1,42 @@
+require './employee.rb'
+
+
+class Department
+  def initialize (department_name)
+    @total_salery = 0
+    @department_name = department_name
+    @employee_array = []
+  end
+
+  def get_department_name
+    return @department_name
+  end
+
+  def add_employee(employee)
+    @employee_array << employee
+  end
+
+#  def output_employee_name
```

Comment:
When you have code that you've commented out, it's best to delete it rather than to leave it in the commits.  You'll always be able to recover it from the git history.